templates/deposit-compliance/index.jsx

```diff
@@ -11,11 +11,26 @@ import {
   CrossRepositoryCheckCard,
 } from './cards'
 
-import { Icon } from 'design'
+import { Icon, Message, Link } from 'design'
 import Title from 'components/title'
 import Markdown from 'components/markdown'
 import { intro as texts } from 'texts/depositing'
 
+const SUPPORT_EMAIL_URL = 'mailto:t%68%65t%65am%40core%2e%61c%2eu%6b'
+const SUPPORT_EMAIL = decodeURIComponent(
+  SUPPORT_EMAIL_URL.slice('mailto:'.length)
+)
```

Comment:
We should move this somewhere in the future. It's hard to think where this should be.

Since it's used only in one place (actually, in 2), I will keep it as is. I hope we will find the proper place for email addresses in the future.